lib/console_web/router.ex

```diff
@@ -40,6 +40,7 @@ defmodule ConsoleWeb.Router do
     post "/devices/delete", DeviceController, :delete
     post "/devices/debug", DeviceController, :debug
     post "/devices/set_active", DeviceController, :set_active
+    get "/events/:device_id", DeviceController, :get_all_events
```

Comment:
maybe change this to /devices/:device_id/events?

Reply:
good point, yeshh